### PR TITLE
[13.0][ADD] account_security_extra_oca

### PR DESCRIPTION
**Pull request by IriaAlonso**
- Change the menu position "Invoicing/Configuration/Payment Mode" to "Invoicing/Accounting/Management/Payment Mode"
- In group Account User adds write and create permission to account.payment.mode 

@dalonsod

### CODE REVIEW

File: account_security_extra_oca/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Account Security Extra",
+    "summary": """
+        - Change the menu position "Invoicing/Configuration/Payment Mode" to "Invoicing/Accounting/Payment Mode"
+        - In group Account User adds write and create permission to account.payment.mode
+    """,
+    "author": "Solvos",
+    "license": "LGPL-3",
```
Comment (dalonsod) [inline account_security_extra_oca/__manifest__.py]: Wrong license (account_payment_mode has AGPL-3, and we have to respect it)

**Comment by IriaAlonso** [comment]
@dalonsod change done

**Review by dalonsod — CHANGES_REQUESTED** [review]
Still missing changes in license in `hooks.py` and generated `README` stuff

**Comment by IriaAlonso** [comment]
@dalonsod done

**Comment by dalonsod** [comment]
Great!
@ChristianSantamaria could you also install this in AluTest enviroment, along with #72 ?

**Comment by ChristianSantamaria** [comment]
Done :+1:

**Comment by dalonsod** [comment]
Major changes made:
* ACL moved from "Show full account features" group to "Invoice" group (see ticket requirements).
* Updated ACL id and fixed typo in ACL name.

**Pull request merged**